test/e2e/cluster_upgrade_runtimesdk_test.go

```diff
@@ -44,7 +44,7 @@ var _ = Describe("When upgrading a workload cluster using ClusterClass with Runt
 		// TODO: We can remove this after the v1.25 release as we then only test the v1.24=>v1.25 upgrade.
 		version, err := semver.ParseTolerant(e2eConfig.GetVariable(KubernetesVersionUpgradeFrom))
 		Expect(err).ToNot(HaveOccurred(), "Invalid argument, KUBERNETES_VERSION_UPGRADE_FROM is not a valid version")
-		if version.LT(semver.MustParse("1.24.0")) {
+		if version.LT(semver.MustParse("1.25.0")) {
```

Comment:
I'm not sure this should be changed 
@sbueringer opinions?

Reply:
Correct. Should stay the same